From a879a035dbdd81898c676c01092b3838ef8a5460 Mon Sep 17 00:00:00 2001
From: Jonas Karlsson <jonas.karlsson@qt.io>
Date: Sun, 1 Jun 2025 23:47:36 +0200
Subject: [PATCH] Remove obsolete <ciso646> header include

This header is obsolete and will not exist in future c++ versions so
remove it now before it causes any issues. We are already getting
warnings in gcc and clang.

Pick-to: 6.10 6.9 6.8
Change-Id: If896592a5f51ac277cc31ea8a776a8704bf63e72
---
 .../PhysX/pxshared/include/foundation/PxPreprocessor.h         | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/3rdparty/PhysX/pxshared/include/foundation/PxPreprocessor.h b/src/3rdparty/PhysX/pxshared/include/foundation/PxPreprocessor.h
index 630c7c0..105e96f 100644
--- a/src/3rdparty/PhysX/pxshared/include/foundation/PxPreprocessor.h
+++ b/src/3rdparty/PhysX/pxshared/include/foundation/PxPreprocessor.h
@@ -31,9 +31,6 @@
 #define PXFOUNDATION_PXPREPROCESSOR_H
 
 #include <stddef.h>
-#if !defined(PX_GENERATE_META_DATA)
-#include <ciso646>  
-#endif
 /** \addtogroup foundation
   @{
 */
-- 
2.49.0

